src/scripts/OSFramework/OSUI/Pattern/OverflowMenu/OverflowMenu.ts

```diff
@@ -0,0 +1,326 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+namespace OSFramework.OSUI.Patterns.OverflowMenu {
+	/**
+	 * Class for the OverflowMenu Pattern
+	 *
+	 * @export
+	 * @class OverflowMenu
+	 * @extends {AbstractPattern<OverflowMenuConfig>}
+	 * @implements {IOverflowMenu}
+	 */
+	export class OverflowMenu extends AbstractPattern<OverflowMenuConfig> implements IOverflowMenu {
+		// Store the aria-label text used on the Trigger Element
+		private _ariaLabelTrigger: string;
+		// Store the Balloon Element
+		private _balloonElem: HTMLElement;
+		// Store the Balloon Class instance
+		private _balloonFeature: Feature.Balloon.IBalloon;
+		// Store the Event Callbacks
+		private _eventBalloonOnToggle: GlobalCallbacks.Generic;
+		private _eventOnClick: GlobalCallbacks.Generic;
+		// Flag used to deal with onBodyClick and open api concurrency methods!
+		private _isOpenedByApi = false;
+		// Store the platform events
+		private _platformEventOnToggle: Callbacks.OSOnToggleEvent;
+		// Store the element that triggers the balloon
+		private _triggerElem: HTMLElement;
+
+		// Store the Balloon options to pass to the Balloon Class
+		public balloonOptions: Feature.Balloon.BalloonOptions;
+		// Store the isOpen ststus
+		public isOpen = false;
+
+		constructor(uniqueId: string, configs: JSON) {
+			super(uniqueId, new OverflowMenuConfig(configs));
+		}
+
+		// Method to handle the custom BalloonOnToggle callback
+		private _balloonOnToggleCallback(_args: string, e: CustomEvent): void {
+			// If the balloon closed is the one from this pattern, toggle the OverflowMenu
+			if (e.detail.balloonElem === this._balloonElem) {
+				this._togglePattern(e.detail.isOpen);
+			}
+		}
+
+		// Method to handle the trigger click callback
+		private _onClickCallback(): void {
+			if (this._balloonFeature.isOpen) {
+				this.close();
+			} else {
+				this._isOpenedByApi = false;
+				this.open(this._isOpenedByApi);
+			}
+		}
+
+		// Method to call the Balloon Class
+		private _setBalloonFeature(): void {
+			this.setBalloonOptions();
+
+			this._balloonFeature = new OSFramework.OSUI.Feature.Balloon.Balloon<IOverflowMenu>(
+				this,
+				this._balloonElem,
+				this.balloonOptions
+			);
+		}
+
+		// Method to handle the Shape config css variable
+		private _setOverflowMenuShape(shape?: GlobalEnum.ShapeTypes): void {
+			if (shape !== undefined) {
+				this.configs.Shape = shape;
+			}
+
+			Helper.Dom.Styles.SetStyleAttribute(
+				this.selfElement,
+				Enum.CssCustomProperties.Shape,
+				'var(--border-radius-' + this.configs.Shape + ')'
```

Comment:
Change this into the ´${}´ notation instead of the concatenation ;)

Reply:
done